File: src/main/java/com/springbootsmini/app/domain/Pet.java
package com.springbootsmini.app.domain;

import java.sql.Timestamp;
import java.util.Date;
import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Setter
@Getter
@NoArgsConstructor
@AllArgsConstructor
public class Pet {
	
	private int petId;
    private String userId;      // user id FK
    private String name;
    private Integer age;
    private Double weight;
    private String gender;
    private Date birthday;
    private String petImage;      // 기본 이미지 나중에 가장 마지막 업데이트된 사진 들고와서 넣기 
    private String content;
    private Timestamp createdAt;

    //여긴 DB에 x 만들기만 
    private List<PetImage> images;       // pet_image 리스트
    private List<PetModule> modules; // pet_weight 리스트

    
}

File: src/main/resources/static/js/pet.js
$(function(){
	
	//펫 상세페이지에서 펫 삭제를 누르면
	$(document).on("click","#deletePet",function(){
	
	let petId = $(this).data("pet-id");
		
	//정말 삭제할거냐고 물어보고 컨트롤러로 보내기 
	if (confirm("정말 삭제할까요?")) {
	    location.href = "/deletePet?petId=" + petId ;
		}
	//아니오 누르면 뭐 걍 암것도 없다 
	});
	
	
	//펫 추가 폼 전부 들어갔는지 확인
	$(document).on("submit","#petAdd",function(){
		
		let name  = $("#name").val();
		let gender  = $("#gender").val();
		let birthday  = $("#birthday").val();
		let age  = $("#age").val();
		let weight  = $("#weight").val();
		let content  = $("#content").val();
		//이미지는 뭐 넣을수도 안넣을 수도
		
		if(name.length == 0) {
							alert("이름을 입력하세요");
							return false;
						}
		
		if(gender.length == 0) {
							alert("성별을 입력하세요");
							return false;
						}				
		if(birthday.length == 0) {
									alert("생일을 입력하세요");
									return false;
								}
				
		if(age.length == 0) {
							alert("나이를 입력하세요");
							return false;
						}
		if(weight.length == 0) {
									alert("몸무게를 입력하세요");
									return false;
								}
				
		if(content.length == 0) {
							alert("특징을 입력하세요");
							return false;
						}						
						
		//다 넣으면 뭐 자기 알아서 submit됨			
	});
	
	
});